src/main/java/com/autotune/common/data/dataSourceMetadata/DataSourceMetadataInfo.java

```diff
@@ -11,42 +9,43 @@
  * DataSourceMetadataInfo represents metadata information about data source(s) and its associated clusters, namespaces, workloads, and containers.
  * It encapsulates a hierarchical structure starting from DataSource to DataSourceContainer, with each nested object containing specific metadata.
  */
-
 public class DataSourceMetadataInfo {
     private static final Logger LOGGER = LoggerFactory.getLogger(DataSourceMetadataInfo.class);
 
     /**
      * Key: Data Source name
      * Value: Associated DataSource object
      */
-    @SerializedName(KruizeConstants.DataSourceConstants.DataSourceMetadataInfoJSONKeys.DATASOURCES)
-    private HashMap<String, DataSource> dataSourceHashMap;
+    //@SerializedName(KruizeConstants.DataSourceConstants.DataSourceMetadataInfoJSONKeys.DATASOURCES)
```

Comment:
why are these commented out?

Reply:
I removed the commented code .. It is commented because the name of attribute is equal to expected serialized json output.